tensorflow_datasets/image/mnist.py

```diff
@@ -218,6 +222,167 @@ def _info(self):
+          name="bymerge",
+          class_number=47,
+          train_examples=697932,
+          test_examples=116323,
+          description="EMNIST ByMerge: 	814,255 characters. 47 unbalanced classes.",
+          version="1.0.1",
+      ),
+      EMNISTConfig(
+          name="balanced",
+          class_number=47,
+          train_examples=112800,
+          test_examples=18800,
+          description="EMNIST Balanced:	131,600 characters. 47 balanced classes.",
+          version="1.0.1",
+      ),
+      EMNISTConfig(
+          name="letters",
+          class_number=37,
+          train_examples=88800,
+          test_examples=14800,
+          description="EMNIST Letters:	103,600 characters. 26 balanced classes.",
+          version="1.0.1",
+      ),
+      EMNISTConfig(
+          name="digits",
+          class_number=10,
+          train_examples=240000,
+          test_examples=40000,
+          description="EMNIST Digits:  280,000 characters. 10 balanced classes.",
+          version="1.0.1",
+      ),
+      EMNISTConfig(
+          name="mnist",
+          class_number=10,
+          train_examples=60000,
+          test_examples=10000,
+          description="EMNIST MNIST:  70,000 characters. 10 balanced classes.",
+          version="1.0.1",
+      ),
+      EMNISTConfig(
+          name="test",
+          class_number=62,
+          train_examples=10,
+          test_examples=2,
+          description="EMNIST test data config.",
+          version="1.0.1",
+      ),
+  ]
+
+  def _info(self):
+      return tfds.core.DatasetInfo(
+          builder=self,
+          description=("The EMNIST dataset is a set of handwritten character digits"
+                       "derived from the NIST Special Database 19  and converted to"
+                       "a 28x28 pixel image format and dataset structure that directly"
+                       "matches the MNIST dataset."
+),
+          features=tfds.features.FeaturesDict({
+              "image": tfds.features.Image(shape=_MNIST_IMAGE_SHAPE),
+              "label": tfds.features.ClassLabel(num_classes=self.builder_config.class_number),
+
+          }),
+          supervised_keys=("image", "label"),
+          urls=["https://www.itl.nist.gov/iaui/vip/cs_links/EMNIST/gzip.zip"],
+          citation=_EMNIST_CITATION,
+      )
+
+  def _split_generators(self, dl_manager):
+
+    filenames = {
+        "train_data": 'emnist-{}-train-images-idx3-ubyte'.format(self.builder_config.name),
+        "train_labels": 'emnist-{}-train-labels-idx1-ubyte'.format(self.builder_config.name),
+        "test_data": 'emnist-{}-test-images-idx3-ubyte'.format(self.builder_config.name),
+        "test_labels": 'emnist-{}-test-labels-idx1-ubyte'.format(self.builder_config.name),
+    }
+
+    dir_name = dl_manager.manual_dir
+
+    if not tf.io.gfile.exists(os.path.join(dir_name, filenames['train_data'])):
+      # The current tfds.core.download_manager is unable to extract multiple and nested files.
```

Comment:
thanks for the explanation - can you create the github issue and reference it here?

Reply:
#234 okay :)